mawabot/cogs/text.py

```diff
@@ -82,6 +83,38 @@ def __init__(self, bot):
 
         await ctx.message.edit(content='\n'.join(lines))
 
+    @commands.command()
+    async def rev(self, ctx, *, text: str):
+        ''' Reverses the text given '''
+
+        await ctx.message.edit(content=text[::-1])
+
+    @commands.command()
+    async def kerrhau(self, ctx, *text: str):
+        ''' "kerrhau"-ifys the given text for meme emphasis '''
+
+        text = list(text)
+        words = []
+
+        while text:
+            rand_num = randint(1, 3)
+            word = []
+
+            try:
+                for _ in range(rand_num):
+                    word.append(text.pop(0))
+            except:
+                pass
```

Comment:
I feel like instead of the bare `except`, simply checking whether `text` still has elements in it would be preferable.

```py
for _ in range(rand_num):
    if text:
        word.append(text.pop(0))
```